src/events/pollVotationResults.js

```diff
@@ -10,11 +10,15 @@ module.exports = {
       return;
     }
 
+    if (!embeds[0]?.fields) {
+      return;
+    }
+
     const userMentionField = embeds[0]?.fields.find(
       (field) => field.name === 'poll_question_text'
     );
     const parts = userMentionField?.value.split('|').map((part) => part.trim());
-    const userMentioned = `<@${parts[1]}>`;
+    const userMentioned = parts?.[1] ? `<@${parts[1]}>` : null;
```

Comment:
You can add another early return here 
```suggestion
    if (!Array.isArray(parts)) return;
    
    const userMentioned = `<@${parts[1]}>`;
```